crates/astria-bridge-withdrawer/src/bridge_withdrawer/ethereum/watcher.rs

```diff
@@ -348,10 +351,21 @@ async fn get_and_forward_block_events(
         .number
         .ok_or_eyre("block did not contain a rollup height")?
         .as_u64();
-    let actions = actions_fetcher
+    let actions: Vec<Action> = actions_fetcher
         .get_for_block_hash(block_hash)
         .await
-        .wrap_err("failed getting actions for block")?;
+        .wrap_err("failed getting actions for block")?
+        .into_iter()
+        .filter_map(|r| {
+            r.map_err(|e| {
+                warn!(
+                    error = &e as &dyn std::error::Error,
```

Comment:
Use our error encoding here:
```suggestion
                    error = %eyre::Report::new(error),
```